tests/runner/test_task_runner.py

```diff
@@ -5,6 +5,7 @@
 import numpy as np
 from pytest_mock import MockerFixture
 import httomo
+from httomo.globals import *
```

Comment:
I'm admittedly not very fond of `*` imports, it's hard to tell what's being brought into scope. Among the formatter changes I can't immediately tell what is being used from `httomo.globals` or where it's being used, could you point me to what's needed? (My guess was `MAX_CPU_SLICES`, but I can't see it anywhere in the additions here?).